### PR TITLE
Creating airflow-postgres-app-services template

### PR DESCRIPTION
**Pull request by HamzaBoukraa**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [X] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog
* Creating airflow-postgres-app-services template

### CODE REVIEW

File: airflow-postgres-app-services/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+                "description": "Name of Airflow web app"
+            }
+        }
+    },
+    "variables": {
+        "databaseName": "airflowdb",
+        "serverName": "[concat(parameters('airflowWebAppName'), '-dbserver-', uniqueString(parameters('guidValue')))]",
+        "AirflowUserName": "[concat(parameters('postgreDatabaseLogin'), '@', variables('serverName'))]",
+        "servicePlan": "[concat(parameters('airflowWebAppName'), '-service-plan-', uniqueString(parameters('guidValue')))]",
+        "pgServerName": "[concat('Microsoft.DBforPostgreSQL/servers/', variables('serverName'))]"
```
Comment (bmoore-msft) [inline airflow-postgres-app-services/azuredeploy.json:147]: remove concat() and use resourceId

File: airflow-postgres-app-services/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+            }
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "Location for all resources."
+            }
+        },
+        "guidValue": {
```
Comment (bmoore-msft) [inline airflow-postgres-app-services/azuredeploy.json:130]: remove this and use an idempotent seed every where you use this (e.g. resourceGroup().id)

File: airflow-postgres-app-services/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+                "name": "[variables('servicePlan')]",
+                "workerSizeId": "1",
+                "reserved": true,
+                "numberOfWorkers": "1"
+            },
+            "sku": {
+                "Tier": "[parameters('servicePlanTier')]",
+                "Name": "[parameters('servicePlanSku')]"
+            },
+            "type": "Microsoft.Web/serverfarms"
```
Comment (bmoore-msft) [inline airflow-postgres-app-services/azuredeploy.json:210]: move this to the top of the resource definition

File: airflow-postgres-app-services/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,263 @@
...
+            "kind": "linux",
+            "location": "[parameters('location')]",
+            "name": "[variables('servicePlan')]",
+            "properties": {
+                "name": "[variables('servicePlan')]",
+                "workerSizeId": "1",
+                "reserved": true,
+                "numberOfWorkers": "1"
+            },
+            "sku": {
```
Comment (bmoore-msft) [inline airflow-postgres-app-services/azuredeploy.json:206]: move this above properties

File: airflow-postgres-app-services/azuredeploy.parameters.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    "airflowWebAppName": {
+      "value": "airflow-postgres-appservices"
+    },
+    "postgreDatabaseLogin": {
+      "value": "pgAdmin"
+    },
+    "postgrePassword": {
+      "value": "pgP@ssw0rd!"
+    },
+    "postgreVersion": {
```
Comment (bmoore-msft) [inline airflow-postgres-app-services/azuredeploy.parameters.json:14]: remove this and use a defaultValue in the template

File: airflow-postgres-app-services/azuredeploy.parameters.json [hunk 6]
```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "airflowWebAppName": {
+      "value": "airflow-postgres-appservices"
```
Comment (bmoore-msft) [inline airflow-postgres-app-services/azuredeploy.parameters.json:6]: remove this and use a defaultValue in the template

File: airflow-postgres-app-services/azuredeploy.parameters.json [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+    },
+    "postgreDatabaseLogin": {
+      "value": "pgAdmin"
```
Comment (bmoore-msft) [inline airflow-postgres-app-services/azuredeploy.parameters.json:9]: GEN-UNIQUE

File: airflow-postgres-app-services/azuredeploy.parameters.json [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+    },
+    "postgrePassword": {
+      "value": "pgP@ssw0rd!"
```
Comment (bmoore-msft) [inline airflow-postgres-app-services/azuredeploy.parameters.json:12]: GEN-PASSWORD

Comment (HamzaBoukraa) [reply airflow-postgres-app-services/azuredeploy.parameters.json]: Updated!

File: airflow-postgres-app-services/azuredeploy.parameters.json (hunk 6 above)
Comment (HamzaBoukraa) [reply airflow-postgres-app-services/azuredeploy.parameters.json]: Updated!

File: airflow-postgres-app-services/azuredeploy.parameters.json (hunk 5 above)
Comment (HamzaBoukraa) [reply airflow-postgres-app-services/azuredeploy.parameters.json]: Updated!

File: airflow-postgres-app-services/azuredeploy.json (hunk 2 above)
Comment (HamzaBoukraa) [reply airflow-postgres-app-services/azuredeploy.json]: Updated!

File: airflow-postgres-app-services/azuredeploy.json (hunk 1 above)
Comment (HamzaBoukraa) [reply airflow-postgres-app-services/azuredeploy.json]: Updated!

File: airflow-postgres-app-services/azuredeploy.json (hunk 3 above)
Comment (HamzaBoukraa) [reply airflow-postgres-app-services/azuredeploy.json]: Updated.

File: airflow-postgres-app-services/azuredeploy.json (hunk 4 above)
Comment (HamzaBoukraa) [reply airflow-postgres-app-services/azuredeploy.json]: Updated.

File: airflow-postgres-app-services/azuredeploy.json [hunk 9]
```diff
@@ -0,0 +1,227 @@
...
+            "type": "Microsoft.Web/serverfarms",
+            "name": "[variables('hostingPlanName')]",
+            "location": "[parameters('location')]",
+            "properties": {
+                "name": "[variables('hostingPlanName')]",
+                "workerSize": "1",
+                "reserved": true,
+                "numberOfWorkers": 1
+            },
+            "sku": {
```
Comment (bmoore-msft) [inline airflow-postgres-app-services/azuredeploy.json:132]: move this above properties

File: airflow-postgres-app-services/azuredeploy.json [hunk 10]
```diff
@@ -0,0 +1,227 @@
...
+                    "type": "config",
+                    "name": "appsettings",
+                    "properties": {
+                        "AIRFLOW__CORE__SQL_ALCHEMY_CONN": "[concat('postgresql://', parameters('administratorLogin'), ':', parameters('administratorLoginPassword'),'@',reference(resourceId('Microsoft.DBforPostgreSQL/servers',variables('serverName'))).fullyQualifiedDomainName,':5432/',variables('databaseName'))]",
+                        "AIRFLOW__CORE__SQL_ALCHEMY_POOL_SIZE": 20,
+                        "AIRFLOW__CORE__LOAD_EXAMPLES": "true",
+                        "AIRFLOW__CORE__EXECUTOR": "LocalExecutor",
+                        "WEBSITES_ENABLE_APP_SERVICE_STORAGE": "true"
+                    },
+                    "tags": {
```
Comment (bmoore-msft) [inline airflow-postgres-app-services/azuredeploy.json:166]: looks like a lot of the resources need to be sorted, see: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties

Comment (HamzaBoukraa) [reply airflow-postgres-app-services/azuredeploy.json]: Updated

File: airflow-postgres-app-services/azuredeploy.json (hunk 9 above)
Comment (HamzaBoukraa) [reply airflow-postgres-app-services/azuredeploy.json]: Updated based on https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties

**Pull request merged**